<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog1</class>
 <widget class="QDialog" name="Dialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="speed1">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>231</y>
     <width>52</width>
     <height>87</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nimbus Sans L</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="speedUnit">
   <property name="geometry">
    <rect>
     <x>636</x>
     <y>324</y>
     <width>57</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nimbus Sans L</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="distance1">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>32</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nimbus Sans L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="distanceUnit">
   <property name="geometry">
    <rect>
     <x>732</x>
     <y>424</y>
     <width>35</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nimbus Sans L</family>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="directionForward">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>390</y>
     <width>74</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="leftSlFlag1">
   <property name="geometry">
    <rect>
     <x>566</x>
     <y>95</y>
     <width>91</width>
     <height>92</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="leftSlFlag2">
   <property name="geometry">
    <rect>
     <x>521</x>
     <y>55</y>
     <width>77</width>
     <height>75</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="rightFlag1">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>95</y>
     <width>91</width>
     <height>92</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="colorLine">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>473</y>
     <width>800</width>
     <height>7</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="roadName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>53</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Droid Arabic Naskh</family>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="speed2">
   <property name="geometry">
    <rect>
     <x>637</x>
     <y>231</y>
     <width>52</width>
     <height>87</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nimbus Sans L</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="speed3">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>231</y>
     <width>52</width>
     <height>87</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nimbus Sans L</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="distance2">
   <property name="geometry">
    <rect>
     <x>632</x>
     <y>400</y>
     <width>32</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nimbus Sans L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="distance3">
   <property name="geometry">
    <rect>
     <x>664</x>
     <y>400</y>
     <width>32</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nimbus Sans L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="distance4">
   <property name="geometry">
    <rect>
     <x>696</x>
     <y>400</y>
     <width>32</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nimbus Sans L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="borderRect">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>400</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>borderRect</zorder>
  <zorder>speed1</zorder>
  <zorder>speedUnit</zorder>
  <zorder>distance1</zorder>
  <zorder>distanceUnit</zorder>
  <zorder>directionForward</zorder>
  <zorder>leftSlFlag2</zorder>
  <zorder>rightFlag1</zorder>
  <zorder>leftSlFlag1</zorder>
  <zorder>colorLine</zorder>
  <zorder>roadName</zorder>
  <zorder>speed2</zorder>
  <zorder>speed3</zorder>
  <zorder>distance2</zorder>
  <zorder>distance3</zorder>
  <zorder>distance4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
